pkg/kv/store.go

```diff
@@ -17,6 +17,8 @@ const (
 	InitialMigrateVersion = 1
 	PathDelimiter         = "/"
 	MetadataPartitionKey  = "kv-internal-metadata"
+
+	ConditionalExists = "dd136a329c1f297d37d039d9ddd99699"
```

Comment:
- We need to define a type - Like `Precondition` or `Precond` (can be string or int - iota) and use `var` to define `ConditionExists` with any value.
- Suggestion name the variable PrecondConditionalExists or something that will include the type / parameter type as a prefix, so it will have more context as the precond value type is interface{}.